diff --git a/node_modules/@react-native-firebase/firestore/lib/index.js b/node_modules/@react-native-firebase/firestore/lib/index.js
index 08a5cfa..1bae251 100644
--- a/node_modules/@react-native-firebase/firestore/lib/index.js
+++ b/node_modules/@react-native-firebase/firestore/lib/index.js
@@ -23,7 +23,7 @@ import {
   isString,
   isUndefined,
   isAndroid,
-} from '@react-native-firebase/app/lib/common';
+} from '@react-native-firebase/app/lib/common/index.js';
 import { setReactNativeModule } from '@react-native-firebase/app/lib/internal/nativeModule';
 import {
   createModuleNamespace,
